Xamarin.Essentials/Platform/Platform.tizen.cs

```diff
@@ -76,11 +77,11 @@ internal static Sensor GetDefaultSensor(SensorType type)
                         Platform.orientationSensor = new TizenOrientationSensor();
                     return Platform.orientationSensor;
                 default:
-                    return null;
+                    throw new InvalidOperationException("switch case should be exhaustive.");
```

Comment:
switch case is currently exhaustive, should not change behaviour and safeguard us against forgetting to amend that switch case and it failing silently